jobs/pgbackup/spec

```diff
@@ -76,3 +76,16 @@ properties:
   pgbackup.s3.endpoint:
     description: "endpoint for backup/restore"
     default: "s3.amazonaws.com"
+
+  pgbackup.s3.proxy.host:
+    description: "Hostname or Ip address of the proxy server, http://your_proxy_server"
```

Comment:
"Hostname or Ip address of the proxy server, http://your_proxy_server. Set an empty string to not configure any proxy (this is the default).